api/src/main/resources/ConfigurationEntry.hbm.xml

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<hibernate-mapping>
+	<class name="org.openmrs.module.patientmatching.ConfigurationEntry" table="patientmatching_configurationEntries">

```

Comment:
Our convention for table names is lowercase with or without underscores as needed to separate words, so the table name `patientmatching_configurationentries` or `patientmatching_configuration_entries` would more closely follow conventions.